website/build-typedoc.sh

```
@@ -1,4 +1,4 @@
-set -e
+set -ex
```

Shellcheck (0.10.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)